src/cache/mod.rs

```diff
@@ -341,6 +351,8 @@ impl Cache {
         if let Some(channel) = self.channels.read().await.get(&id) {
             let channel = channel.clone();
             return Some(Channel::Guild(channel));
+        } else if let Some(channel) = self.temp_channels.get_if_present(&id) {
```

Comment:
I renamed `get_if_present` method to `get`. Sorry for the last minutes change.

```suggestion
        } else if let Some(channel) = self.temp_channels.get(&id) {
```